website/content/en/docs/Installation/_index.md

```diff
@@ -43,3 +47,14 @@ kubectl get pods -w
 ```
 
 Now **Jenkins Operator** should be up and running in the `default` namespace.
+
+### Using Helm Chart
+
+There is a option to use Helm to install the operator. It requires the Helm 3+ for deployment.
+
+To install, you need only to type these commands:
```

Comment:
To install, you need only to type these commands: -> The chart can then be installed directly: